### PR TITLE
Power-management lxqt config

### PR DESCRIPTION
**Pull request by meets2tarun**
### Short description
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
This PR will take care of switching of screen saver and power management in LXQt version of meilix.
#88 
### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by tabesin** [comment]
Pls rebase

**Comment by tabesin** [comment]
Please rebase, rebuild metapackage, so we can merge.

File: build.sh [hunk 1]
```diff
@@ -112,6 +112,11 @@ apt-get -q -y -o Dpkg::Options::="--force-overwrite" --purge install chromium-br
 #rm -rf /etc/chromium-browser
 mv /etc/chromium-browser_ /etc/chromium-browser
 
+#set the screen-saver and power-management off
```
Comment (meets2tarun) [inline build.sh]: Yes, these changes will go in .profile not here.

**Comment by meets2tarun** [comment]
this will solve this issue https://github.com/fossasia/meilix/pull/136

**Pull request closed without merging**